src/gui/widgets/Fader.cpp

```diff
@@ -61,75 +62,50 @@ namespace lmms::gui
 
 SimpleTextFloat * Fader::s_textFloat = nullptr;
 
-Fader::Fader( FloatModel * _model, const QString & _name, QWidget * _parent ) :
-	QWidget( _parent ),
-	FloatModelView( _model, this ),
-	m_fPeakValue_L( 0.0 ),
-	m_fPeakValue_R( 0.0 ),
-	m_persistentPeak_L( 0.0 ),
-	m_persistentPeak_R( 0.0 ),
-	m_fMinPeak( 0.01f ),
-	m_fMaxPeak( 1.1 ),
-	m_back(embed::getIconPixmap("fader_background")),
-	m_leds(embed::getIconPixmap("fader_leds")),
+Fader::Fader(FloatModel* model, const QString& name, QWidget* parent) :
```

Comment:
If we set the defaults in `Fader.h` instead then we don't need to have this huge initialization list.

Reply:
Having just read [this discussion](https://stackoverflow.com/questions/28413154/c-initializing-variables-in-header-vs-with-constructor) about the pros and cons I think I'd rather keep it in the implementation file.

Reply:
Can you explain more of your opinion?

Reply:
Mostly the argument on having to potentially recompile lots of sources when a value is changed. It somehow feels like the default values should be an implementation detail.

On the other hand, if the values were defined in the header files, then looking at the header would suffice for a developer to know, if the default values are as expected or whether an instance has to be potentially reconfigured after construction.

I think there is no 100% correct way.

Reply:
> Mostly the argument on having to potentially recompile lots of sources when a value is changed. It somehow feels like the default values should be an implementation detail.

How often are we changing the default values? Also, giving the defaults in the class is part of the [core guidelines](https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#c45-dont-define-a-default-constructor-that-only-initializes-data-members-use-default-member-initializers-instead) not once [but twice](https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#c48-prefer-default-member-initializers-to-member-initializers-in-constructors-for-constant-initializers).

Reply:
Ok, you have convinced me! :sweat_smile: Adjusted with commit ad8a31daa37.